script/results.html

```diff
@@ -62,6 +62,11 @@
   <button class="w3-button w3-right w3-light-grey" id="download" type="submit" onclick=download()
     hidden="hidden">Download Results Feedback</button>
 
+    <button class="w3-button w3-right w3-light-grey" id="downloadjson" type="submit" onclick=downloadjson()
+    hidden="hidden">Download Feedback as a JSON file</button>
+    <p hidden="hidden" id ="UploadFeedback">
+      Upload Feedback File: <input type="file" id="feedbackFile" size="40">
```

Comment:
The upload seems to work only the first time for me, after this there is no effect